client/src/components/utils/appHelper.js

```diff
@@ -117,6 +167,14 @@ const setOriginToCurrentLocation = (props) => {
   }
 };
 
+/**
+ * Get directions.
+ * @param  {Object}   props            Props passed into the App component
+ * @param  {Object}   interactionTypes Available immutable interaction types
+ * @param  {Object}   place            Origin or destination place information
+ * @param  {Function} dispatchFunction Dispatcher that is passed into the action
```

Comment:
Wording is a little bit weird here because the dispatcher dispatches an action.

I followed through the code and this parameter is actually the dispatcher itself. I think the description should say something along the lines of: "Dispatcher for updating redux store"